src/theme/components/checkbox.ts

```diff
@@ -8,16 +11,27 @@ import {
   checkboxRadioHoverStyles,
 } from "./global";
 
+// This function creates a set of function that helps us
+// create multipart component styles.
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers([
+    "helperErrorText",
+    "icon",
+    "control",
+    "label",
+    "container",
+  ]);
+
 // Style object for the Checkbox's visual icon.
-const baseStyleIcon = {
-  transitionProperty: "transform",
-  transitionDuration: "normal",
+const baseStyleIcon = defineStyle({
+  // Remove Chakra's default animation
+  animation: null,
```

Comment:
Is this something we need to watchout for in Chakra v2?

Reply:
For some components, yes, but the animation is obvious enough to make us check if there are any custom animations we need to remove.